core/new-gui/src/app/workspace/component/left-panel/operator-menu/operator-menu.component.ts

```diff
@@ -24,14 +24,14 @@ import { NzAutocompleteOptionComponent } from "ng-zorro-antd/auto-complete";
 @UntilDestroy()
 @Component({
   selector: "texera-operator-panel",
-  templateUrl: "./operator-menu-frame.component.html",
-  styleUrls: ["./operator-menu-frame.component.scss"],
+  templateUrl: "operator-menu.component.html",
+  styleUrls: ["operator-menu.component.scss"],
   providers: [
     // uncomment this line for manual testing without opening backend server
     // { provide: OperatorMetadataService, useClass: StubOperatorMetadataService }
   ],
 })
-export class OperatorMenuFrameComponent implements OnInit {
+export class OperatorMenuComponent implements OnInit {
```

Comment:
if you rename it, make sure you also update the comments and docs (and file names, packages, etc). For example, line 13 has a mention of this class. Please fix other instances as well.

Reply:
Fixed.